tox.ini

```diff
@@ -1,5 +1,8 @@
 [tox]
 envlist = tests, flake8
+toxworkdir=/tmp/tox
```

Comment:
Running tests through tox required extra configuration in tox.ini as we are now running with uid 10001 as per the [dockerflow](https://github.com/mozilla-services/Dockerflow) spec and write permission has been tightened up in the Dockerfile.